shell/browser/api/electron_api_printing.cc

```diff
@@ -28,8 +28,6 @@ struct Converter<printing::PrinterBasicInfo> {
     dict.Set("name", val.printer_name);
     dict.Set("displayName", val.display_name);
     dict.Set("description", val.printer_description);
-    dict.Set("status", val.printer_status);
```

Comment:
This should also be removed from https://github.com/electron/electron/blob/3d2a7545314f68c21f3fb92fb0a6eb9edcceabd3/docs/api/structures/printer-info.md#L22-L23

Reply:
Removed in 58456376d425b298a8551aa6fe94f1835291c734